data/drak/drak.txt

```diff
@@ -143,6 +143,8 @@ outfit "Drak Antimatter Fragment"
 		"inaccuracy" 1
 		"lifetime" 499
 		"hit force" 1000
+		"disruption damage" 50
+		"heat damage" 250
```

Comment:
This heat damage is hardly anything. A plasma cannon has nearly five times this DPS.

Reply:
OK. I'll increase it.